index.d.ts

```diff
@@ -0,0 +1,11 @@
+import { Model, ModelClass } from 'objection';
+
+declare module 'objection-graphql-relay' {
+  export function idWrapper(modelName?: string): (parent: Model) => Promise<string>;
+  export function fromGlobalId(modelName?: string, globalId?: string): string;
+  export function singleRelationshipWrapper(relationshipName: string): Promise<string>;
+  export function connectionWrapper(field: string | object): Promise<object>;
+  export function pagedRelationQuery(instance: ModelClass<Model>, field: string | object, first: number, after: number, extraFilter?: (builder: any) => object, orderBy?: (builder: any) => object): Promise<object>;
```

Comment:
`builder` is an instance of `Objection.QueryBuilder` - The return value is also `Promise<ConnectionWrapperResults>` instead of `Promise<object>` (you can use the `ConnectionWrapperResults` type from above).